modules/src/main/modules/search/elasticsearch/module.xml

```diff
@@ -18,5 +18,9 @@
         <module name="org.hibernate.commons-annotations"/>
         <module name="org.jboss.logging"/>
         <module name="org.apache.httpcomponents"/>
+
+        <!-- AWS authentication support -->
+        <!-- Note: we must export the services (hence the classes), because services are loaded using the -engine classloader -->
+        <module name="org.hibernate.search.elasticsearch-aws" export="true" services="export" optional="true" slot="${hibernate.search.version}" />
```

Comment:
I understand the need for `services="export"`. But are you sure that `export="true"` was also necessary?